tests/clients/test_dapr_grpc_client.py

```diff
@@ -42,19 +42,24 @@
 
 
 class DaprGrpcClientTests(unittest.TestCase):
-    server_port = 8080
+    grpc_port = 50001
+    http_port = 3500
     scheme = ''
     error = None
 
-    def setUp(self):
-        self._fake_dapr_server = FakeDaprSidecar()
-        self._fake_dapr_server.start(self.server_port)
+    @classmethod
+    def setUpClass(cls):
```

Comment:
I moved the server creation in all tests to the `setUpClass` method. With this, we got a speed improvement of around 300% (from ~6m to ~1.45m).